Helper/Data.php

```diff
@@ -49,8 +59,7 @@ public function isOverrideNotification()
 
     public function enableLog()
     {
-        $enableLog = $this->settings->enableLog();
-        return $enableLog;
+        return $this->settings->enableLog();
     }
```

Comment:
Address on future refactor (not now): 
`enableLog()` here is to get the value of whether merchant enable logging, right? current func name is a bit confusing, because usually calling `enableSomething()` is to perform action of enabling something, not getting a value.

Possible fix:
- rename function to `getIsLogEnabled`. Or
- rename the private var to `isLogEnabled`, so that the func name become `isLogEnabled()`

Reply:
this method is no longer used. It will deprecated and remove on the next release